connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndReadITSuite.scala

```diff
@@ -163,6 +165,32 @@ class SparkBigQueryEndToEndReadITSuite extends FunSuite
 
   def testsWithReadInFormat(dataSourceFormat: String, dataFormat: String): Unit = {
 
+    test(("testing view with different columns for select and filter. " +
+      "DataSource %s. Data Format %s").format(dataSourceFormat, dataFormat)) {
+
+      for(i <- 1 to 2){
```

Comment:
can you please move the DF creation to a helper method and remove the loop in order to improve readability?

Reply:
done